diff --git a/vllm/platforms/rocm.py b/vllm/platforms/rocm.py
index 9072c71426..693d4d5392 100644
--- a/vllm/platforms/rocm.py
+++ b/vllm/platforms/rocm.py
@@ -414,7 +414,7 @@ def use_rocm_custom_paged_attention(
             and (qtype == torch.half or qtype == torch.bfloat16)
             and head_size == 128
             and block_size == 16
-            and (gqa_ratio >= 3 and gqa_ratio <= 16)
+            and (gqa_ratio >= 1 and gqa_ratio <= 16)
             and max_seq_len <= 128 * 1024
             and alibi_slopes is None
             and kv_cache_dtype == "auto"
